docs/faqs.md

```diff
@@ -177,6 +177,17 @@ No. Some projects provide more than a simple library, but also applications. For
 When a non standard open-source license is used, we have decided to use `LicenseRef-` as a prefix, followed by the name of the file which contains a custom license.
 See [the reviewing guidlines](reviewing.md#license-attribute) for more details.
 
+## How do I flag a problem to a recipe consumer?
+
+Regardless of why, if the recipe detects a problem where binaries might not be generated correctly, an exception must be raised. This to prevent the publishing
+incorrect packages which do not work as intented. Use `ConanInvalidConfiguration` which is specially support in ConanCenter.
+
+```py
+raise ConanInvalidConfiguration(f"The project {self.name}/{self.version} requires liba.enable_feature=True.")
```

Comment:
```suggestion
raise ConanInvalidConfiguration(f"The project {self.ref} requires liba.enable_feature=True.")
```

self.ref == self.name/self.version

Reply:
Sweet. But i copy pasted this from somewhere else 🙈